src/instruments/src/PFD/LandingSystemIndicator.tsx

```diff
@@ -13,6 +14,18 @@ export class LandingSystem extends DisplayComponent<{ bus: EventBus, instrument:
 
     private deviationGroup = FSComponent.createRef<SVGGElement>();
 
+    private roll = Arinc429Word.empty();
+
+    private pitch = Arinc429Word.empty();
+
+    private handleGsReferenceLine() {
+        if (this.lsButtonPressedVisibility || (this.pitch.isNormalOperation() && this.roll.isNormalOperation())) {
```

Comment:
After a little bit of research, it turns out it should be shown when the altitude is valid, not the attitude. Source: https://youtu.be/bzKOdsusuQo?t=249
Only alt is valid: 
![grafik](https://user-images.githubusercontent.com/39596827/172082698-0cff1af0-2eb5-4e7e-8f8c-b258039d5b68.png)
A few moments later, now att is valid too: 
![grafik](https://user-images.githubusercontent.com/39596827/172082744-30431216-d8a4-41de-b11a-6016402684d5.png)
It can be seen that the yellow bar is visible in both images, so it has to be visible as long as altitude is valid.

Reply:
Fixed 👍